### PR TITLE
replace base64 -d with base64 --decode

### PR DESCRIPTION
**Pull request by StevenTobin**
## Additional Information
To allow an installation to be run from MacOS base64 -d should be base64 --decode

## Verification Steps

* git checkout this branch
* Run a successful installation

## Is an upgrade task required and are there additional steps needed to test this?

- [x] Tested Installation
- [x] Tested Uninstallation
- [ ] Tested or Created follow on for Upgrade

### CODE REVIEW

**Comment by StevenTobin** [comment]
@Nanyte25 could you try this to validate that it fixes the issues you were seeing?

File: roles/3scale/tasks/admin_token.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 - name: Retrieve 3Scale admin auth token
-  shell: oc get secret/system-seed -n {{ threescale_namespace }} -o template --template=\{\{.data.ADMIN_ACCESS_TOKEN\}\} | base64 -d
```
Comment (jessesarn) [inline roles/3scale/tasks/admin_token.yml]: Might be better to do the base64decode right in the template.   i.e.
```
oc get secret/system-seed -n 3scale -o template --template='{{.data.ADMIN_ACCESS_TOKEN | base64decode}}'
```

File: roles/prerequisites/tasks/upgrade.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@
   when: not (run_master_tasks | default(true) | bool)
 
 - name: Get target version
-  shell: "oc get secret manifest -n {{ eval_webapp_namespace }} -o json | jq \".data.generated_manifest\" -r | base64 -d | jq \".version\" -r"
```
Comment (jessesarn) [inline roles/prerequisites/tasks/upgrade.yml]: Similar here:
```
oc get secret manifest -n webapp -o template='{{.data.generated_manifest | base64decode}}' | jq ".version" -r
```

File: scripts/get_admin_secret.sh [hunk 3]
```diff
@@ -1,3 +1,3 @@
 #!/usr/bin/env bash
 
-oc get secret admin-user-credentials --template='{{index .data "password"}}' -n sso | base64 -d
```
Comment (jessesarn) [inline scripts/get_admin_secret.sh]: 1. looks like this doesn't take ns_prefix into account
2. can remove call to external base64 binary with:
```
oc get secret admin-user-credentials --template='{{index .data "password" | base64decode}}' -n sso
```

**Pull request closed without merging**